### ISSUE TITLE
Rename the single sign-on for enterprise accounts

### ISSUE THREAD

**Issue by alachaum** [KeypDemoCorp/awesome-ui#155]
We should add logging so we can debug this more easily in the future.

Accessibility should be considered when implementing this.

The current implementation does not scale well beyond a few thousand records.

**Comment by pixdemo2** [comment]
Linking this to the work the platform team is doing this sprint.

Agreed on the approach, let's write up the plan.
